contents/data-display/charts/date-range-temperature-charts/metadata.json

```
@@ -0,0 +1,18 @@
+{
+  "en": {
+    "title": "Date Range Temperature Charts",
```

There isn't multiple charts, so it should be `Chart`.
```suggestion
    "title": "Date Range Temperature Chart",
```